tests/regression/anomaly/test_anomaly_segmentation.py

```diff
@@ -0,0 +1,238 @@
+"""Tests for Anomaly Segmentation with OTX CLI."""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+import os
+import random
+from pathlib import Path
+from timeit import default_timer as timer
+
+import pytest
+
+from otx.cli.registry import Registry
+from tests.regression.regression_test_helpers import (
+    ANOMALY_DATASET_CATEGORIES,
+    TIME_LOG,
+    get_result_dict,
+    get_template_performance,
+    load_regression_configuration,
+)
+from tests.test_suite.e2e_test_system import e2e_pytest_component
+from tests.test_suite.run_test_command import (
+    nncf_eval_testing,
+    nncf_optimize_testing,
+    otx_deploy_openvino_testing,
+    otx_eval_compare,
+    otx_eval_deployment_testing,
+    otx_eval_e2e_eval_time,
+    otx_eval_e2e_train_time,
+    otx_eval_openvino_testing,
+    otx_export_testing,
+    otx_train_testing,
+    pot_eval_testing,
+    pot_optimize_testing,
+)
+
+# Configurations for regression test.
+TASK_TYPE = "anomaly_segmentation"
```

Comment:
Might be quite a lot of refactor for this PR but since the difference between the three tests is the task_type, how about we use pytest.parameterize and just keep a single file?

Reply:
Good point. However, it needs lots of changes in the current PR. 
As you know, although many parts of this PR need refactoring, we should get the regression / KPI test results until the next release. 

So, I want to suggest to postpone the refactoring process after merging this.